cluster/manifests/skipper/deployment.yaml

```diff
@@ -93,6 +93,12 @@ spec:
 {{ if or (eq .ConfigItems.nlb_switch "pre") (eq .ConfigItems.nlb_switch "exec") }}
         - name: HTTP_REDIRECT
           value: "true"
+{{ end }}
+{{ if eq .ConfigItems.skipper_lua_scripts_enabled "true" }}
+        - name: LUA_PATH
+          value: /etc/skipper/lua/?.lua
+        - name: DATADOME_API_KEY
+          value: "{{ .ConfigItems.datadome_api_key }}"
```

Comment:
This should have its own `if`

Reply:
why? no

Reply:
You may have lua scripts without datadome_api_key

Reply:
but then an empty string datadome api key, does not matter, too